src/shared/dumb_components/field.js

```diff
@@ -11,23 +11,23 @@ var Field = (props) =>
   ? <Input {...props} />
   : <div></div>
 
-var Display = ({ name, id, value, description, memberView }) =>
-  <p>
-    {description && name === 'News type: ' && <span><br /><i>{description}</i><br /></span>}
-    {memberView && <br />}
-    <span className='info'>{name}</span>
-    <span id={'view-member-' + id}>
-      {caser(id, value)}
-    </span>
-  </p>
+var Display = ({ name, value, description, userViewClass }) =>
```

Comment:
I thought that we might need to know if it was a class for styling on admin view or on member view. Not sure if that makes sense if this is a display component. Have changed it back.